docs/conf.py

```diff
@@ -203,4 +203,5 @@
 # Ref: https://stackoverflow.com/a/30624034/595220
 nitpick_ignore = [
     # temporarily listed ('role', 'reference') pairs that Sphinx cannot resolve
+    ('py:class', 'GenericOptionalPrimitiveType'),
```

Comment:
Let's not do ignores that can be avoided. This creates todolists that nobody tracks or fixes over time. It's easier to just not add tech debt instead.